config/config.go

```diff
@@ -15,15 +15,16 @@ const (
 	EnvIgnoreDaemonSets      = "IGNORE_DAEMON_SETS"
 	EnvDeleteLocalData       = "DELETE_LOCAL_DATA"
 	EnvClusterName           = "CLUSTER_NAME"
+	EnvAutodiscoveryTags     = "AUTODISCOVERY_TAGS"
 	EnvAutoScalingGroupNames = "AUTO_SCALING_GROUP_NAMES"
 	EnvAwsRegion             = "AWS_REGION"
 )
 
 type config struct {
 	Environment           string   // Optional
 	Debug                 bool     // Defaults to false
-	AutoScalingGroupNames []string // Required if ClusterName not provided
-	ClusterName           string   // Required if AutoScalingGroupNames not provided
+	AutoScalingGroupNames []string // Required if ClusterName or AutoScalingGroupNames not provided
```

Comment:
Since you're deleting `ClusterName`, we should probably update the comment accordingly

Reply:
Fixed